### PR TITLE
Updates per 2019-06-19 meeting

### PR DESCRIPTION
**Pull request by GeoffreyBooth**
Look I took notes! (Sorta.)

### CODE REVIEW

File: doc/plan-for-new-modules-implementation.md [hunk 1]
```diff
@@ -99,25 +99,24 @@ Phase 3 improves user experience and extends the MVP. Phase 3 is malleable based
   - Should loaders be per package, per application or either?
   - **Status**: In development.
 
-* Map the paths within modules, providing similar functionality as the browser’s [import maps proposal](https://github.com/WICG/import-maps#packages-via-trailing-slashes).
+* `"exports"` field: for consumers of a package, map the paths of deep imports to provide encapsulation (an explicit public API); pretty specifiers (no file exensions or paths that include things like `dist/`) and flexibility for future package versions renaming or moving files without affecting the package’s public API. Applies to both ESM and CommonJS.
+  - Proposal: [Package Exports Proposal](https://github.com/jkrems/proposal-pkg-exports).
+  - **Status**: Ready for PR.
+
+* `"imports"` field: provide mappings for `import` specifiers within a package, similar to browsers’ [import maps proposal](https://github.com/WICG/import-maps#packages-via-trailing-slashes).
```
Comment (MylesBorins) [inline doc/plan-for-new-modules-implementation.md]: this is not accurate per the conversation in the meeting. I think we discussed exports as being something we had consensus on moving for with whereas imports was something to be further discussed

Comment (GeoffreyBooth) [reply doc/plan-for-new-modules-implementation.md]: Yes, and isn’t that what I wrote?

Comment (MylesBorins) [reply doc/plan-for-new-modules-implementation.md]: To be very clear, I think it is premature to include this proposal at this time

Comment (GeoffreyBooth) [reply doc/plan-for-new-modules-implementation.md]: I can remove `"imports"`, I guess that’s what you’re saying? Though I didn’t think we had such a high bar for adding things to this doc? We already have a status saying that it’s nowhere close to consensus.

Comment (MylesBorins) [reply doc/plan-for-new-modules-implementation.md]: We have had a high bar for removing things from the doc and the doc has been pointed to as a source of truth regarding things we are exploring. I would prefer we don't add it for now

**Pull request merged**